src/main/plugins/com.sophos.tocjs/integrator.xml

```diff
@@ -29,7 +32,7 @@ See the accompanying LICENSE file for applicable license.
             <isset property="output.file" />
         </not>
     </condition>
-    <property name="output.file" value="${output.dir}${file.separator}toctree.js"/>
+    <property name="output.file" value="${ditaot.render.dir}${file.separator}toctree.js"/>
```

Comment:
The `ditaot.` prefix for the property doesn't follow any of the assorted conventions we currently have. The only real consistent naming scheme we have is prefixing internal properties with `dita.`. It's not really good, but I don't want to introduce `ditaot.` since with would be yet another convention.